README.md

```diff
@@ -4,7 +4,8 @@
     - [Table of Contents](#table-of-contents)
     - [Community](#community)
     - [Deploying StackRox](#deploying-stackrox)
-        - [Installation via Helm](#installation-via-helm)
+        - [Quick Installation via Helm](#quick-installation-via-helm)
+        - [Manual Installation via Helm](#manual-installation-via-helm)
```

Comment:
I know it was like this, but I would replace `via` with `using` :)

Reply:
Right, avoid latin